.github/workflows/testing.yml

```diff
@@ -63,3 +65,58 @@ jobs:
         run: |
           cd app/
           python manage.py check
+  lighthouse:
+    runs-on: ubuntu-latest # operating system your code will run on
+    services:
+      postgres:
+        image: postgres:16
+        env:
+          POSTGRES_USER: sadilar
+          POSTGRES_PASSWORD: sadilar
+          POSTGRES_DB: term_db
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-test.txt
+          sudo apt-get install -y gettext
+          npm install -g @lhci/cli@0.14.x
+      - name: Setup LHCI files (env & entrypoint)
+        run: chmod +x entrypoint.sh && cp .env.lhci app/.env
```

Comment:
Please fix the `chmod` in the repo directly so that everybody has it as executable. Then this first part can be removed.